charts/drone/values.schema.json

```
@@ -66,6 +67,24 @@
       "$id": "#/properties/securityContext",
       "type": "object"
     },
+    "serviceAccount": {
+      "$id": "#/properties/serviceAccount",
+      "type": "object",
+	  "required": [
+		  "name",
+		  "create"
+	  ],
+	  "properties": {
+		  "name": {
+			  "$id": "#/properties/serviceAccount/name",
+			  "type": "string"
+		  },
+		  "create": {
+			  "$id": "#/properties/serviceAccount/create",
+			  "type": "boolean"
+		  }
+	  } 
+    },
```

whitespacing is inconsistent. You've got tabs and a trailing space.
```suggestion
    "serviceAccount": {
      "$id": "#/properties/serviceAccount",
      "type": "object",
      "required": [
        "name",
        "create"
      ],
      "properties": {
        "name": {
          "$id": "#/properties/serviceAccount/name",
          "type": "string"
        },
        "create": {
          "$id": "#/properties/serviceAccount/create",
          "type": "boolean"
        }
      }
    },
```